chatapplication/src/components/palette.css

```diff
@@ -0,0 +1,117 @@
+:root {
```

Comment:
Cool stuff. I would say organize this file a little bit better.
After every variable group, leave a space to know what's happening.